components/GoalsCover.js

```diff
@@ -257,6 +261,16 @@ class GoalsCover extends React.Component {
           isReached: true,
         }),
       );
+      goals.push(
+        this.createGoal('monthlyBudget', {
+          animateProgress: true,
+          title: intl.formatMessage(this.messages['bar.monthlyBudget']),
+          amount: get(collective, 'stats.yearlyBudget') / 12,
+          precision: 0,
+          position: 'below',
+          isReached: true,
+        }),
+      );
```

Comment:
We have responsiveness issues with this graph, we should limit as much as possible the number of items we show on it. I suggest doing something like:

```es6
if (hasMonthlyGoal(...)) createGoal('monthlyBudget');
else createGoal('yearlyBudget');
```